README.md

```diff
@@ -1,44 +1,46 @@
-# (Inofficial) Foundry VTT Types
+# foundry-vtt-types
+
+> TypeScript definitions for [Foundry VTT](https://foundryvtt.com/) (unofficial)
 
 <div align=center>
 
-[![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
 [![League Discord Server](https://img.shields.io/discord/732325252788387980?label=League%20of%20Extraordinary%20Foundry%20VTT%20Developers)](https://discord.gg/52DNPzqm2Z)  
-![GitHub issues by-label](https://img.shields.io/github/issues-raw/kmoschcau/foundry-vtt-types/foundry%200.7.9?color=%23fe631d)
-![GitHub closed issues by-label](https://img.shields.io/github/issues-closed-raw/kmoschcau/foundry-vtt-types/foundry%200.7.9?color=%23fe631d)
+![GitHub issues by-label](https://img.shields.io/github/issues-raw/League-of-Foundry-Developers/foundry-vtt-types/foundry%200.7.9?color=%23fe631d)
+![GitHub closed issues by-label](https://img.shields.io/github/issues-closed-raw/League-of-Foundry-Developers/foundry-vtt-types/foundry%200.7.9?color=%23fe631d)
 
 </div>
 
-## Introduction
-
-This was originally created by @NickEastNL for his Project Creator over on
-[GitLab](https://gitlab.com/foundry-projects/foundry-pc/foundry-pc-types). This
-fork is an attempt to update the types to 0.7.9 of Foundry and to provide better
-maintenance.
+## Installation
 
-## Installing
+With [npm](https://npmjs.org/) installed, run
 
 ```bash
-npm install --save-dev github:kmoschcau/foundry-vtt-types
+$ npm install --save-dev github:League-of-Foundry-Developers/foundry-vtt-types
```

Comment:
Personally, I don't like adding $ or similar in the beginning. That prevents you from simply copy-pasting the complete command by selecting it with a tripple click.

Reply:
Yeah I agree with ghost here. It should be clear from the context, that this should be run in a shell.

Reply:
Updated 👍🏼